derive/src/attributes/row.rs

```diff
@@ -0,0 +1,43 @@
+use quote::ToTokens;
+
+pub const ATTRIBUTE_NAME: &str = "row";
+pub const ATTRIBUTE_SYNTAX: &str = "#[row(crate = ...)]";
+
+pub const CRATE_PATH: &str = "crate";
+pub const DEFAULT_CRATE_PATH: &str = "::clickhouse";
+
+pub struct Row {
+    pub crate_path: syn::Path,
+}
+
+impl Default for Row {
+    fn default() -> Self {
+        let default_crate_path = syn::parse_str::<syn::Path>(DEFAULT_CRATE_PATH).unwrap();
+        Self {
+            crate_path: default_crate_path,
+        }
+    }
+}
+
+impl<'a> TryFrom<&'a syn::Attribute> for Row {
+    type Error = &'a syn::Attribute;
+
+    fn try_from(attr: &'a syn::Attribute) -> Result<Self, Self::Error> {
+        if attr.path().is_ident(ATTRIBUTE_NAME) {
```

Comment:
Not very important (because it's a private API), but it's more straightforward to return an error (even a simple string) and panic in the `From` instance.